storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1826,6 +1827,9 @@ bool xb_get_one_option(int optid, const struct my_option *opt, char *argument) {
     case OPT_XTRA_ENCRYPT_KEY:
       hide_option(argument, &xtrabackup_encrypt_key);
       break;
+
+#include "sslopt-case.h"
+
```

Comment:
include in middle of file, intentional?? :thinking:

Reply:
yes, this file contains the `case OPT_` from SSL options. I've added it on the end of the block, right before print help following the location where client binaries do - https://github.com/percona/percona-server/blob/8.0/client/mysql.cc#L2086